sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -223,18 +223,18 @@ IList<ServiceBusMessage> messages = new List<ServiceBusMessage>();
 messages.Add(new ServiceBusMessage(Encoding.UTF8.GetBytes("First")));
 messages.Add(new ServiceBusMessage(Encoding.UTF8.GetBytes("Second")));
 // send the messages
-await sender.SendAsync(messages);
+await sender.SendMessagesAsync(messages);
```

Comment:
We need to update the same [here ](https://github.com/Azure/azure-sdk-for-net/blob/332e4375d0805bbe016a3cce5cbb8ebe93f03e23/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md#sending-messages).

Reply:
Also, need to update the method names in the description.